xrv9k/docker/launch.py

```diff
@@ -73,21 +73,34 @@ def __init__(self, hostname, username, password, nics, conn_mode, vcpu, ram, ins
     def gen_mgmt(self):
         """Generate qemu args for the mgmt interface(s)"""
         res = []
-        # mgmt interface
-        res.extend(
-            ["-device", "virtio-net-pci,netdev=mgmt,mac=%s" % vrnetlab.gen_mac(0)]
+        
+        
+        self.mgmt_mac = (
+            "c0:00:01:00:ca:fe"
+            if getattr(self, "_static_mgmt_mac", False)
+            else vrnetlab.gen_mac(0)
```

Comment:
I think this is the only spot I think we should change here.
The reason originally we overridden the `gen_mgmt` function of the parent class was that we needed to add two more dummy interfaces.

But I think instead of redefining the whole method, we should call `super().get_mgmt` to get the skeleton with all the forwarded ports and so on, and then just add two dummy to the list